src/Compilers/CSharp/Test/Emit3/FirstClassSpanTests.cs

```diff
@@ -3352,6 +3542,136 @@ static class C
         CompileAndVerify(comp, expectedOutput: "11").VerifyDiagnostics();
     }
 
+    [Theory, MemberData(nameof(LangVersions))]
+    public void OverloadResolution_ReadOnlySpanVsArray_06(LanguageVersion langVersion)
+    {
+        var source = """
+            using System;
+
+            C.M(default(object[]));
```

Comment:
> C.M(default(object[])); [](http://example.com/codeflow?start=11&length=24)

Minor: Consider adding this case to the previous test rather than creating a separate test.